src/contract/assembled_transaction.ts

```diff
@@ -793,15 +780,15 @@ export class AssembledTransaction<T> {
    * currently supported!
    */
   signAuthEntries = async ({
-    expiration = this.getStorageExpiration(),
+    expiration = (async () =>
+      (await this.server.getLatestLedger()).sequence + 100)(),
```

Comment:
100 ledgers covers ~500 seconds which ~8.3 minutes, let's add a comment here with a liiittle more specific instead of the broader "5 to 10"

Reply:
Updated the comment